ldap/servers/plugins/pwdstorage/pwd_init.c

```diff
@@ -52,6 +52,8 @@ static Slapi_PluginDesc smd5_pdesc = {"smd5-password-storage-scheme", VENDOR, DS
 
 static Slapi_PluginDesc pbkdf2_sha256_pdesc = {"pbkdf2-sha256-password-storage-scheme", VENDOR, DS_PACKAGE_VERSION, "Salted PBKDF2 SHA256 hash algorithm (PBKDF2_SHA256)"};
 
+static Slapi_PluginDesc gost_yescrypt_pdesc = {"gost-yescrypt-password-storage-scheme", VENDOR, DS_PACKAGE_VERSION, "Yescrypt algorithm (STREEBOG256)"};
```

Comment:
Can this description be extended to "Yescrypt KDF algorithm", just to make it clear that this is a KDF, and that it is appropriate for modern crypto usage.